SOURCES/nexus.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Sonatype Nexus Repository Manager
+After=network.target
+
+[Service]
+Type=forking
+ExecStart=/usr/share/nexus/bin/nexus start
+ExecStop=/usr/share/nexus/bin/nexus stop
+User=nexus
+Restart=on-abort
+#Environment=INSTALL4J_JAVA_HOME=/foo/bar/jre8
```

Comment:
I don't think we need the variable. If at some time we decide to allow the user to change this, we should do it somewhere else (maybe /etc/default/nexus).

Reply:
Nexus3 and nexus work without INSTALL4J_JAVA_HOME.

Reply:
Yes, go on for both nexus and nexus3.

Reply:
Deleted Environment=INSTALL4J_JAVA_HOME=/foo/bar/jre8